README.md

```diff
@@ -145,91 +220,54 @@ kurtosis enclave inspect YOUR_ENCLAVE_ID_HERE
 
-```javascript
-const ethers = require("ethers")
 ```
-
-Now let's get a connection to the node with service ID `bootnode` by getting a [JsonRpcProvider](https://docs.ethers.io/v5/api/providers/jsonrpc-provider/). First we'll grab the `ServiceContext` (Kurtosis' representation of the node), use it to get the ports that the bootnode is listening on, and then construct an Ethers client connected to the node:
-
-```javascript
-// Grab the bootnode's service context
-bootnodeServiceId = executeEthModuleResultObj.bootnode_service_id
-bootnodeNodeObj = executeEthModuleResultObj.node_info[bootnodeServiceId]
-getBootnodeServiceCtxResult = await enclaveCtx.getServiceContext(bootnodeServiceId)
-bootnodeServiceCtx = getBootnodeServiceCtxResult.value
-
-// Get the IP & port of the bootnode, inside the enclave
-bootnodeRpcPortId = bootnodeNodeObj.rpc_port_id
-bootnodeRpcPort = bootnodeServiceCtx.getPrivatePorts().get(bootnodeRpcPortId)
-bootnodePrivateIp = bootnodeServiceCtx.getPrivateIPAddress()
-
-// Instantiate the Ethers client
-bootnodeRpcProvider = new ethers.providers.JsonRpcProvider(`http://${bootnodePrivateIp}:${bootnodeRpcPort.number}`)
+curl -X POST HTTP_URL_HERE --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":83}' -H "Content-Type: application/json"
 ```
 
-Notice how we used the `executeEthModuleResultObj` object containing details about the Ethereum network, which we got from executing the module at the very beginning.
-
-Finally, let's verify that our Ethereum network is producing blocks:
+And you will see something like this, where the result is a hexadecimal number representing the current block number:
 
-```javascript
-blockNumber = await bootnodeRpcProvider.getBlockNumber()
-if (blockNumber > 0) { console.log("All is well!"); }
+```
+{"jsonrpc":"2.0","id":83,"result":"0x93"}
 ```
 
 And that's it! Anything doable in Ethers is now doable against your private Ethereum network running in your Kurtosis enclave.
 
-To exit out of the REPL you can enter any of:
-
-* Ctrl-D
-* Ctrl-C, twice
-* `.exit`
-
-Your enclave will stay running until you command it to stop. Run the following to tell Kurtosis to tear down your enclave, replacing `YOUR_ENCLAVE_ID` with the ID of your running enclave:
+To clean up all the recent elements created you can execute the following:
 
 ```
-kurtosis enclave rm -f YOUR_ENCLAVE_ID
+kurtosis clean -a
 ```
 
+and Kurtosis will tear down all enclaves and everything inside.
+
 Step Five: Get An Ethereum Testsuite (5 minutes)
 ---------------------------------------------
-Manually verifying against a sandbox network is nice, but it'd be great if we could take our logic and run it as part of CI. Fortunately, Kurtosis allows us to do exactly that. 
+Manually verifying against an enclave is nice, but it'd be great if we could take our logic and run it as part of CI. Kurtosis has a testing framework that allows us to do exactly that.
```

Comment:
s/Kurtosis has a testing framework that allows us to do exactly that/Fortunately, Kurtosis was designed from the beginning with testing in mind./